projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -192,4 +194,8 @@ export class PdfViewerComponent implements AfterContentInit, OnChanges, OnDestro
       }), 0);
     }
   }
+
+  toggleCommentsSummary() {
+    this.toolbarEvents.toggleCommentsSummary(!this.toolbarEvents.showCommentSummary.getValue());
```

Comment:
as above, I think this could be done in the header component, it would save us having to bubble the event up to the parent component

Reply:
sorry, I disagree because I think the header component should be a stateless component. it the same used on image and pdf view so i rather do separate stuff  in the parent level then have ngif on the component level if image if pdf which is  not really nice